### PR TITLE
Fix #175: Add testcase for multibyte LikeHandler

### LINKED ISSUE #175: Find a test case with multibyte strings for `Like` filter handler
https://github.com/yiisoft/data/blob/6c4a3a8fb3b1bd8de4dd051e86c8f7c6e2140f0b/src/Reader/Iterable/FilterHandler/LikeHandler.php#L33-L36

### PR DESCRIPTION
**Pull request by samdark**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

**Review by arogachev — APPROVED** [review]

**Review by vjik — CHANGES_REQUESTED** [review]
Need a line in changelog

**Comment by samdark** [comment]
There's no change for end user at all so I doubt changelog is needed.

**Comment by samdark** [comment]
Added CHANGELOG.

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 - New #176: Add `OrderHelper` (@vjik)
 - New #173, #184: Add `$caseSensitive` parameter to `Like` filter to control whether the search must be case-sensitive
   or not (@arogachev)
+- Chg #190: Add testcase for multibyte `LikeHandler`, use `str_contains` for case-sensitive match (@samdark)
```
Comment (vjik) [inline CHANGELOG.md:30]:
```suggestion
- Enh #190: Use `str_contains` for case-sensitive match in `LikeHandler` (@samdark) 
```

**Review by vjik — APPROVED** [review]

**Pull request merged**